docs/_config.yml

```diff
@@ -22,6 +22,9 @@ menu:
   -
     name: o2-tutorial.html
     label: TUTORIAL (O2)
+  -
+    name: o2-daq-tutorial.html
+    label: TUTORIAL (O2 DAQ)
```

Comment:
It applies to all O2 DAQ packages. `flpproto`, as explained in the doc, is the "example" package (and by the way at some point it will be renamed to something else according to the authors).